multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -72,6 +72,28 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Price (Dollars) to bid for spot instances. Omit for on-demand instances.
 # workerSpotPrice: "0.05"
 
+## Etcd Cluster config
```

Comment:
Probably need documentation going over how the nodes will be distributed across subnets / how that logic works. E.g. we recommend X nodes, with Y subnets, and it will be distributed in this way.